### PR TITLE
Lutaml `diagram` syntax

### PR DESCRIPTION
**Pull request by w00lf**
lutaml/lutaml-uml#37 Support for `diagram` tag attribute lutaml syntax, rubocop fixes for lib/lutaml/uml/parsers/dsl.rb file

### CODE REVIEW

File: lib/lutaml/uml/document.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@
 module Lutaml
   module Uml
     class Document
+      class UknownMemberTypeError < StandardError; end
```
Comment (ronaldtse) [inline lib/lutaml/uml/document.rb:8]: "Unknown"

File: lib/lutaml/uml/class.rb [hunk 2]
```diff
@@ -9,7 +9,8 @@ module Uml
     class Class < Classifier
       class UknownMemberTypeError < StandardError; end
```
Comment (ronaldtse) [inline lib/lutaml/uml/class.rb:10]: Can we differentiate between Class::UknownMemberTypeError and Document::UknownMemberTypeError with different names?

Comment (w00lf) [reply lib/lutaml/uml/class.rb:10]: Sure, will do. I have deleted it for now though, we don't need it for now.

File: lib/lutaml/uml/document.rb (hunk 1 above)
Comment (w00lf) [reply lib/lutaml/uml/document.rb:8]: Fixed

**Pull request merged**